apollo-environment/src/main/java/com/spotify/apollo/environment/ApolloEnvironmentModule.java

```diff
@@ -65,8 +65,14 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ApolloEnvironmentModule.class);
 
-  public static ApolloEnvironmentModule create() {
```

Comment:
Default creator was removed. It should still be left but cause the default "random" ordering behaviour? Or are we making this a breaking change?